.flake8

```diff
@@ -1,3 +1,6 @@
 [flake8]
 exclude = sematic/ui/node_modules
 max-line-length=90
+per-file-ignores =
+    ./sematic/examples/add/pipeline.py: W291
```

Comment:
Honestly, I don't know the best way to go forward: 

Using two trailing white-spaces is a good way to denote a line break in MarkDown. But `flake8` doesn't like it (W291 is `W291 trailing whitespace`). And `flake8` doesn't provide a way to do ignore a code block...

Unfortunately I had to end up making those two files exceptions.